tests/SprykerSdkTest/Zed/AppSdk/Communication/Console/CreateManifestConsoleTest.php

```diff
@@ -32,12 +33,44 @@ class CreateManifestConsoleTest extends Unit
      */
     public function testCreateManifestConsole(): void
     {
+        // Arrange
         $commandTester = $this->tester->getConsoleTester(new CreateManifestConsole());
 
+        $filePath = sprintf('%s/config/app/manifest/', vfsStream::setup('root')->url());
+
         // Act
-        $commandTester->execute([CreateManifestConsole::MANIFEST_NAME => 'Manifest', '--' . CreateManifestConsole::OPTION_MANIFEST_PATH => 'config/example/en_US.json'], ['verbosity' => OutputInterface::VERBOSITY_VERBOSE]);
+        $commandTester->execute(
+            [
+                CreateManifestConsole::MANIFEST_NAME => 'Manifest',
+                CreateManifestConsole::MANIFEST_LOCALE => 'en_US',
+                '--' . CreateManifestConsole::OPTION_MANIFEST_PATH => $filePath,
+            ],
+            ['verbosity' => OutputInterface::VERBOSITY_VERBOSE],
+        );
```

Comment:
```suggestion
        $commandTester->execute(
            [
                CreateManifestConsole::MANIFEST_NAME => 'Manifest',
                CreateManifestConsole::MANIFEST_LOCALE => 'en_US',
            ],
            ['verbosity' => OutputInterface::VERBOSITY_VERBOSE],
        );
```

Reply:
When not passing a path the file will already be written in the virtual filesystem. The whole test suite is set up like this.

Reply:
If I remove this line  '--'. CreateManifestConsole::OPTION_MANIFEST_PATH => $filePath,
then execute command will create new file on file system because it took default file path and then our test case failed on second time.
So we need to pass virtual file path here. Please suggest.